source/preprocessor.tex

```diff
@@ -271,14 +271,12 @@
 
 \pnum
 Preprocessing directives of the forms
-
 \begin{ncbnftab}
 \indextext{\idxcode{\#if}}%
-\terminal{\# if}\>\>constant-expression new-line group\opt\br
+\terminal{\# if} constant-expression new-line group\opt\br
 \indextext{\idxcode{\#elif}}%
-\terminal{\# elif}\>\>constant-expression new-line group\opt
```

Comment:
This is a tab stop in a tabular setting. But I figured that that was completely unnecessary here (and I have also changed the tab stops to fit better with the main grammars, so this wasn't quite working anymore.)